### PR TITLE
Add PHP yoda conditions section

### PR DESCRIPTION
**Pull request by nunorafaelrocha**
This adds Yoda conditions section to the PHP coding standards.

R= @fixe , @ruimarinho , @nunofgs , @franciscocardoso .

### CODE REVIEW

File: php/README.md [hunk 1]
```diff
@@ -147,3 +148,17 @@ use OtherVendor\OtherPackage\Qux;
 
 // ...
 ```
+
+## Yoda Conditions
+
+You *must* use yoda conditions when doing logical comparisons `===` and `!==`. Avoid using yoda conditions for `<`, `>`, `<=` and `>=` since they are difficult to read.
```
Comment (fixe) [inline php/README.md]: @ruimarinho I thought we were ditching Yoda completely?

Comment (ruimarinho) [inline php/README.md]: What does Symfony use?

Comment (nunorafaelrocha) [inline php/README.md]: Although they don't say to use yoda conditions, their examples have yoda conditions. For example, in the [Symfony's coding standards](http://symfony.com/doc/current/contributing/code/standards.html) page, you can see `if ('string' === $dummy)`.

Comment (ruimarinho) [inline php/README.md]: Maybe we should say "must not use yoda conditions for.." (and an example too).

**Comment by nunorafaelrocha** [comment]
@ruimarinho Done :)

**Pull request merged**